.github/gen-workflow-ci.py

```diff
@@ -635,14 +635,14 @@ def publish_docker_images(needs: List[str], images: List[str]) -> str:
                 f'            type=semver,pattern={{{{major}}}}\n'
                 f'            type=sha\n'
                 f'\n'
-                f'      - name: Set up QEMU\n'
-                f'        uses: docker/setup-qemu-action@v1\n'
```

Comment:
Just out of curiosity: Was QEMU ever needed before?

Reply:
It does not look like it ever was.